r/R/csv.R

```diff
@@ -500,7 +500,7 @@ CsvWriteOptions$create <- function(include_header = TRUE, batch_size = 1024L, nu
   )
 }
 
-readr_to_csv_read_options <- function(skip = 0, col_names = TRUE, col_types = NULL) {
+readr_to_csv_read_options <- function(skip = 0, col_names = TRUE) {
```

Comment:
This fixes a bug where passing the `col_types` argument into `open_dataset` didn't work.  This parameter shouldn't be here, as whilst `col_names` is a CsvReadOptions option, `col_types` is a `CsvConvertOptions` option anyway.